lib/hammer_cli_foreman/hosts/common_update_options.rb

```diff
@@ -50,9 +50,15 @@ def self.ask_password
         HammerCLI.interactive_output.ask(prompt) { |q| q.echo = false }
       end
 
+      def self.set_image_uuid(params, image_id)
```

Comment:
How can I load cr-id from the server? You can have multiple compute resource on the servers

Reply:
It would be just for a case of an update of an existing host. We could try to load the host and check if the CR is set there. If not we should rise usage/validation error or similar